www/src/_layouts/docs.njk

```diff
@@ -6,10 +6,10 @@ styles:
 
 <div class="docs">
   <nav class="docs__primary-nav">
-    <div class="docs__primary-nav__content flex-row">
```

Comment:
Sorry to see you `flex-go` 😢 
(it's for the best though. It tripped me up a couple times scanning the styles)